ts/features/wallet/onboarding/cobadge/saga/networking/searchUserCobadge.tsx

```diff
@@ -18,15 +18,14 @@ type CoBadgeRequestQuery = {
 };
 
 const cobadgePrefix = "WALLET_ONBOARDING_COBADGE_SEARCH";
-const privativePrefix = "WALLET_ONBOARDING_PRIVATIVE_SEARCH";
 
 const withTokenSuffix = "WITH_TOKEN";
 const withoutTokenSuffix = "WITHOUT_TOKEN";
 
 /**
- * This method contains the generic networking logic used to request a user's cobadge and privative
- * cards. The request for privative cards also requires the pancode
- * @param cobadgeQuery  the input data to search the privative / cobadge
+ * This method contains the generic networking logic used to request a user's cobadge
+ * { and privative cards. The request for privative cards also requires the pancode ** DEPRECATED **}
+ * @param cobadgeQuery  the input data to search the {privative} / cobadge
```

Comment:
```suggestion
 * This method contains the generic networking logic used to request a user's cobadge
 * @param cobadgeQuery  the input data to search the cobadge
```

Reply:
adressed in https://github.com/pagopa/io-app/commit/b9230048e2ad74c789fb95c5efbdf376e86a5186